controllers/bootstrap/init.go

```diff
@@ -372,6 +372,11 @@ func (b *Bootstrap) InitResources(instance *apiv3.CommonService) error {
 			klog.Errorf("Failed to create control namespace: %v", err)
 			return err
 		}
+		klog.Info("Creating OperatorGroup for IBM Common Services in control namespace")
+		if err := b.CreateOperatorGroup(b.CSData.ControlNs); err != nil {
```

Comment:
Yes, it is for shared instance mode.

The first place in `main.go` is used by cs-operator installed cloud pak namespace, where it is only a bootstrap to create master and control namespace.
The second place in `init.go` is used by cs-operator installed CS namespace. In case, CloudPak and CS are deployed in the same namespace, and bootstrap is also the functional operator.